examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -673,6 +683,15 @@ void wfx_rsi_task(void * arg)
                 {
                     wfx_ipv6_notify(GET_IPV6_SUCCESS);
                     hasNotifiedIPV6 = true;
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+                    // enabling the power save mode for RS9116 if sleepy device is enabled
+                    rsi_status = rsi_wlan_power_save_profile(RSI_SLEEP_MODE_2, RSI_MAX_PSP);
+                    if (rsi_status != RSI_SUCCESS)
+                    {
+                        WFX_RSI_LOG("Powersave Config Failed, Error Code : 0x%lX", rsi_status);
+                    }
+                    WFX_RSI_LOG("Powersave Config Success");
+#endif /* CHIP_DEVICE_CONFIG_ENABLE_SED */
```

Comment:
why is the same code copy/pasted twice?

Reply:
it was added twice due to the ipv4 and ipv6 
fixed on the latest commit having a function calling this interaction for power save